policies/rules.md

```diff
@@ -146,6 +146,8 @@ To participate, team members must be at least 13 years of ago at the time the co
 
 Prizes amounts and types will be determined at the discretion of organizers and event sponsors, and distributed in a manner they see fit.
 
+In the event of a tie for a prize awarded by vote, then the winner will be deemed the first project to receive their final vote. If this does not resolve the tie then the organizers reserve the right to determine a winner based on their judgement. The organizers will strive to do so in an appropriate, fair and impartial manner.
```

Comment:
>If this does not resolve the tie then the 

I'm not an English major, but this looks like it could use a comma instead of "then" to break up the run-on:
> If this does not resolve the tie, the

Reply:
Changed to a comma.